spec/forms/support_interface/region_form_spec.rb

```diff
@@ -153,4 +153,94 @@
       it { is_expected.to be_valid }
     end
   end
+
+  describe "#save" do
+    subject(:save!) { form.save! }
+
+    let(:form) do
+      described_class.new(
+        region:,
+        teaching_authority_online_checker_url: "",
+        teaching_authority_name:,
+        teaching_authority_emails_string:,
+        teaching_authority_requires_submission_email: false,
+        all_sections_necessary: true,
+        requires_preliminary_check: false,
+        sanction_check: "none",
+        status_check: "none",
+        teaching_authority_provides_written_statement: false,
+        written_statement_optional: true,
+        teaching_authority_websites_string:,
+        teaching_authority_address: "",
+        other_information: "",
+        teaching_authority_certificate: "",
+        status_information: "",
+        sanction_information: "",
+        teaching_qualification_information: "",
+      )
+    end
+
+    let(:region) { create(:region) }
+    let(:teaching_authority_name) { "Teaching Authority" }
+    let(:teaching_authority_emails_string) do
+      "email1@example.com\nemail2@example.com"
+    end
+    let(:teaching_authority_websites_string) do
+      "http://site1.com\nhttp://site2.com"
+    end
+
+    context "when form is valid" do
+      it "updates region attributes" do
+        subject
+        expect(region).to have_attributes(
+          application_form_skip_work_history: false,
+          other_information: "",
+          reduced_evidence_accepted: false,
+          requires_preliminary_check: false,
+          sanction_check: "none",
+          sanction_information: "",
+          status_check: "none",
+          status_information: "",
+          teaching_authority_address: "",
+          teaching_authority_certificate: "",
+          teaching_authority_emails: %w[email1@example.com email2@example.com],
+          teaching_authority_name: "Teaching Authority",
+          teaching_authority_online_checker_url: "",
+          teaching_authority_provides_written_statement: false,
+          teaching_authority_requires_submission_email: false,
+          teaching_authority_websites: %w[http://site1.com http://site2.com],
+          teaching_qualification_information: "",
+          written_statement_optional: true,
+        )
+      end
+
+      it "processes emails correctly" do
+        subject
+        expect(region.teaching_authority_emails).to eq(
+          %w[email1@example.com email2@example.com],
+        )
+      end
```

Comment:
You already have these in the `updates region attributes` test, don't think you need to repeat them.

Reply:
ah yes, I wrote these first and then found I had to change the previous test to include the email rather than "" to get the test to pass.